src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -197,6 +197,10 @@ protected function addFlash($type, $message)
             throw new \LogicException('You can not use the addFlash method if sessions are disabled. Enable them in "config/packages/framework.yaml".');
         }
 
+        if (!method_exists($this->container->get('session'), 'getFlashBag')) {
```

Comment:
`$this->container->get('session')` is now being done twice, can you put the result of this method call in a variable? That will reduce the performance impact of searching for it twice

Reply:
Indeed. Fixed :slightly_smiling_face: